src/melee/it/it_803F1418.c

```diff
@@ -0,0 +1,1204 @@
+bool it_802934AC();
+bool it_802934D0();
+bool it_802934F4();
+bool it_80293514();
+void it_80293534();
+void it_80293640();
+void it_8029367C();
+bool it_802939E0();
+void it_802939E8();
+void it_80293A08();
+bool it_80293D6C();
+void it_80293D74();
+void it_80293F58();
+void it_802940FC();
+void it_802941F8();
+void it_802942B0();
+void it_80294344();
+void it_8029451C();
+void it_80294728();
+void it_80294790();
+void it_80294A28();
+void it_80294A90();
+void it_80294AB0();
+void it_80294BE4();
+void it_80294C14();
+void it_80294C3C();
+void it_80294CE4();
+bool it_80294D4C();
+bool it_80294D54();
+bool it_80294D5C();
+bool it_80294D7C();
+bool it_80294DA0();
+void it_80295118();
+void it_8029524C();
+void it_80295298();
+void it_80295340();
+void it_802953A8();
+void it_802953C8();
+void it_80295524();
+void it_80295554();
+bool it_802955E0();
+void it_80295684();
+void it_802956EC();
+void it_8029570C();
+void it_802958C8();
+void it_80295908();
+void it_80295974();
+bool it_80295ACC();
+void it_80295B08();
+bool it_80295B70();
+bool it_80295BAC();
+bool it_80295BCC();
+bool it_80295C08();
+void it_80295C28();
+void it_80295C48();
+void it_80295D90();
+void it_80295DC0();
+void it_80295E4C();
+void it_80295EB4();
+void it_802960E0();
+void it_80296120();
+void it_8029683C();
+void it_802968D8();
+void it_80296930();
+int it_8029789C();
+int it_802978A4();
+int it_802978AC();
+int it_802978B4();
+void it_802978BC();
+void it_802978DC();
+void it_80297904();
+void it_80297ABC();
+void it_80297AF0();
+void it_80297B18();
+bool it_80297BB8();
+void it_80297BF8();
+bool it_80298060();
+bool it_802980A0();
+bool it_802980D4();
+bool it_80298114();
+void it_80298148();
+bool it_80298634();
+bool it_8029863C();
+bool it_80298644();
+bool it_8029864C();
+bool it_80298654();
+bool it_80298828();
+void it_802988C4();
+bool it_80298BE4();
+bool it_80298BEC();
+bool it_80298BF4();
+bool it_80298BFC();
+bool it_80298C04();
+bool it_80298CE8();
+void it_80298DCC();
+bool it_80299360();
+bool it_80299368();
+bool it_80299370();
+bool it_80299378();
+bool it_80299380();
+bool it_802993A0();
+void it_802993C0();
+bool it_80299B6C();
+bool it_80299B74();
+bool it_80299C08();
+bool it_80299C10();
+bool it_80299C18();
+bool it_80299C20();
+void it_80299C28();
+void it_80299D4C();
+void it_80299DD0();
+void it_80299E28();
+void it_80299E48();
+bool it_8029A0B0();
+bool it_8029A0B8();
+bool it_8029A0C0();
+bool it_8029A0C8();
+bool it_8029A0EC();
+void it_8029A0F4();
+bool it_8029A600();
+bool it_8029A608();
+bool it_8029A610();
+bool it_8029A618();
+bool it_8029A620();
+bool it_8029A704();
+void it_8029A728();
+void it_8029A9FC();
+void it_8029AD1C();
+void it_8029AD94();
+void it_8029AF34();
+void it_8029B0A8();
+void it_8029B188();
+void it_8029B1AC();
+void it_8029B2F4();
+void it_8029B328();
+void it_8029B348();
+bool it_8029B46C();
+bool it_8029B4AC();
+bool it_8029B4EC();
+bool it_8029B52C();
+bool it_8029B56C();
+void it_8029B670();
+void it_8029B6D8();
```

Comment:
Are these only used in the tables, or are they called/defined elsewhere? Please use `(void)` instead of `()`.

Reply:
They are defined in each item's TU, and take a Item_GObj*. These are just temporary, they will be moved to the individual items' headers as those items are matched.